docs/writing-tests/testdriver.md

```diff
@@ -185,6 +185,27 @@ await test_driver.set_permission({ name: "background-fetch" }, "denied");
 await test_driver.set_permission({ name: "push", userVisibleOnly: true }, "granted", true);
 ```
 
+### set_spc_transaction_mode
+
+Usage: `test_driver.set_spc_transaction_mode(mode, context=null)`
+  * _mode_: a [TransactionAutomationMode](https://w3c.github.io/secure-payment-confirmation/#enumdef-transactionautomationmode)
+    that describes the mode to set SPC into
+  * _context_: a WindowProxy for the browsing context in which to perform the call
+
+This function places [Secure Payment Confirmation](https://w3c.github.io/secure-payment-confirmation/)
+into an automated 'autoaccept' or 'autoreject' mode, to allow testing without
+user interaction with the transaction UX prompt. It returns a promise that
+resolves after the transaction mode has been set to _mode_.
+
+Example:
+
+``` js
+await test_driver.set_spc_transaction_mode("autoaccept");
+// Assumption: `request` is a PaymentRequest with a secure-payment-confirmation
+// payment method.
+const response = request.show();
+```
```

Comment:
Examples tend to get copied by other developers, so they should follow best practices. Here, would it be better to reset to "none" after the response is received?

Reply:
Done, thanks.